### DISCUSSION TITLE
nodriver tab.save_screenshot() does not use the "full_page" parameter

### DISCUSSION THREAD

**Discussion by pkooiman** [ultrafunkamsterdam/undetected-chromedriver#1829] [General]
If I understand correctly, due to line 1212
`cdp.page.capture_screenshot(format_=format, capture_beyond_viewport=True)`
save_screenshot will always capture the complete viewport regardless of the full_page parameter.
I managed to work around it by passing the cdp directly instead of using tab.save_screenshot, seems to work OK.

**Comment by ultrafunkamsterdam** [comment]
I'm aware of that. Being fixed in coming update.

**Reply by pkooiman** [reply]
Great, just getting started with nodriver and it’s amazingly fast and easy to use!

**Reply by dsmabulage** [reply]
> I'm aware of that. Being fixed in coming update.

Is this fixed
